### DISCUSSION TITLE
Execute Python code in WSL

### DISCUSSION THREAD

**Discussion by sfiruch** [Microsoft/PTVS#7431] [Enchancements]
**Describe the feature you are requesting**
VS should be able to run Python code in WSL, similar to the functionality available for C++ developers (https://learn.microsoft.com/en-us/cpp/build/walkthrough-build-debug-wsl2?view=msvc-170). I'd like to run and debug my code running in WSL.

**Reasons for requesting this feature**
I work on a Python project which uses a lot of native code that is linux-x64 only. I'd like to use VS as IDE to work on this project.

**Describe alternatives you've considered**
- Porting all library dependencies of my python project to windows infeasible
- Using [remote debugging](https://learn.microsoft.com/en-us/visualstudio/python/debugging-python-code-on-remote-linux-machines?view=vs-2022) is cumbersome and not suitable as daily development workflow.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
